block/manager.go

```diff
@@ -610,7 +600,7 @@ func (m *Manager) publishBlock(ctx context.Context) error {
 		if err != nil {
 			return nil
 		}
-		block.SignedHeader.Header.NextAggregatorsHash = m.getNextAggregatorsHash()
+		//block.SignedHeader.Header.NextAggregatorsHash = m.getNextAggregatorsHash()
```

Comment:
In general, we should avoid adding commented code, and just remove it. 

If we're commenting code, then I'm assuming we plan to add this back at some point? If that is the case, then most of this PR should be commenting code out and not removing code at all with a follow-up issue to add back commented code with relevant documentation.

Reply:
yes i know but you wanted to review before it was ready haha

Reply:
Gabriel marked it to ready for review lol